public/js/searchBar.js

```diff
@@ -0,0 +1,60 @@
+import { URL } from './util/data.js';
+import { $searchBar, $searchBarInput, $searchSuggestions, $rollingKeywords, $recommendedKeyword } from './util/ref.js';
+import { CLASS_LIST } from './util/cssClasses.js';
+
+class SearchBar {
+  constructor() {
+    this.ui = $searchBar;
+    this.inputArea = $searchBarInput;
+    this.suggestions = $searchSuggestions;
+  }
+
+  registerEvent() {
```

Comment:
같은 맥락에서 `CLASS_LIST`를 외부에서 인자로 받는 것이 조금 더 괜찮아 보이지만 `CLASS_LIST`가 자주 변경될 것 같지는 않기 때문에 지금 정도의 의존성도 괜찮은 것 같아요. 어떤 쪽으로 수정하라고 제안드리는 것은 아니고, 클래스의 의존성에 항상 주의할 필요하 있다는 말씀을 드리고 싶었어요.
```suggestion
registerEvent(CLASS_LIST) {
}
```

Reply:
감사합니다!! 인자로 넘겨주는 연습을 많이 해보겠습니다 😊